src/index.html

```diff
@@ -7,12 +7,76 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <a
+        class="card__link-img"
+        href="#product"
+      >
+        <img
+          class="card__img"
+          src="./images/imac.jpeg"
+          alt="Product image"
+        />
+      </a>
+
+      <a
+        class="card__link-description"
+        href="#product"
+      >
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </a>
+
+      <div class="card__code">
+        Product code:
+        <span class="card__code-value">195434</span>
+      </div>
+
+      <div class="card__rating">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+        <div class="card__rating-reviews">Reviews: 5</div>
+      </div>
+
+      <div class="card__price">
+        Price
+        <span class="card__price-value">$2,199</span>
+      </div>
```

Comment:
Also, you can use some semantic tags, for example paragraph - `p` for texts

Reply:
Changed